### ISSUE TITLE
Typo in run-vault options "--agent-ca-cert-file" vs "--agent-ca-cert_file"

### ISSUE THREAD

**Issue by jholloway7** [hashicorp/terraform-aws-vault#191]
In the run-vault script, there seems to be an inconsistency in the `agent-ca-cert-file` flag. 

It is documented here as "agent-ca-cert-file": 

https://github.com/hashicorp/terraform-aws-vault/blob/b2a673c8998db1906fb18fbc86ae9bad899b702c/modules/run-vault/run-vault#L55

But the switch statement here references it as `agent-ca-cert_file`:

https://github.com/hashicorp/terraform-aws-vault/blob/b2a673c8998db1906fb18fbc86ae9bad899b702c/modules/run-vault/run-vault#L567

Based on the labeling of the other flags, I believe the former is correct and the latter is a typo. 

I'd be happy to submit a PR for this issue, but curious if the typo should continue to be supported in addition to the corrected version until the next major release since changing it might break users who have found the workaround.

**Comment by brikis98** [comment]
Good catch. The latter is indeed a typo. It should be `--agent-ca-cert-file`. A PR to fix this would be very welcome! Supporting the old version, and adding a comment to explain it's for backwards compatibility, sounds like a good idea.
